templates/tools/doxygen/Doxyfile.php.include

```diff
@@ -0,0 +1,25 @@
+
+<%namespace import="doxy_base" file="Doxyfile.base"/>
+<%def name="gen_doxyfile(internal)">
+<%
+  import itertools
+  import glob
+  import os
+  import fnmatch
+  srcdoc = []
+  for dirpath, dirnames, filenames in os.walk('src/php'):
+    for filename in filenames:
+      if os.path.splitext(filename)[1] == '.md':
```

Comment:
Why do we need the `.md` files?

Reply:
referenced other languages. It shows in related pages.